kunta-api-server/src/test/java/fi/otavanopisto/kuntaapi/test/AbstractIntegrationTest.java

```diff
@@ -0,0 +1,132 @@
+package fi.otavanopisto.kuntaapi.test;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.io.IOUtils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.client.WireMock;
+
+public abstract class AbstractIntegrationTest extends AbstractTest {
+
+  public class AbstractMocker {
+    
+    private List<StringGetMock> stringMocks;
+    private List<BinaryGetMock> binaryMocks;
+    
+    public AbstractMocker() {
+      stringMocks = new ArrayList<>();
+      binaryMocks = new ArrayList<>();
+    }
+    
+    public void mockGetBinary(String path, String type, String binaryFile) {
+      try (InputStream binaryStream = getClass().getClassLoader().getResourceAsStream(binaryFile)) {
+        binaryMocks.add(new BinaryGetMock(path, type, IOUtils.toByteArray(binaryStream)));
+      } catch (IOException e) {
+        fail(e.getMessage());
+      }
+    }
+    
+    public void mockGetString(String path, String type, String content) {
+      stringMocks.add(new StringGetMock(path, type, content));
+    }
+    
+    public void mockGetJSON(String path, Object object) {
+      try {
+        stringMocks.add(new StringGetMock(path, "application/json", new ObjectMapper().writeValueAsString(object)));
+      } catch (JsonProcessingException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)